### ISSUE TITLE
No compiled binaries for 1.3.0?

### ISSUE THREAD

**Issue by jonaz** [danielqsj/kafka_exporter#218]
They seems to be missing for 1.3.0?

**Comment by nantiferov** [comment]
Interesting, no binaries for 1.3.0 while docker image is built.
So far it's possible to get `linux/amd64` binaries from there:

```
$ docker run --rm -it --entrypoint /bin/sh -v $PWD:/opt danielqsj/kafka-exporter:v1.3.0
/ # cp /bin/kafka_exporter /opt/
/ # exit
$ ls
kafka_exporter
```

**Comment by jonaz** [comment]
ping @danielqsj

**Comment by danielqsj** [comment]
I have fixed it in 1.3.1.
